src/main/java/com/emc/mongoose/base/control/run/RunServlet.java

```diff
@@ -83,13 +83,15 @@ protected final void doPost(final HttpServletRequest req, final HttpServletRespo
 			scenarioPart = null;
 		}
 		try {
-			final var defaults = mergeIncomingWithLocalConfig(defaultsPart, resp, aggregatedConfigWithArgs);
+			final var config = mergeIncomingWithLocalConfig(defaultsPart, resp, aggregatedConfigWithArgs);
 			final var scenario = getIncomingScenarioOrDefault(scenarioPart, appHomePath);
-
+			if (config.stringVal("run-id") == null) {
+				config.val("run-id", LogUtil.getDateTimeStamp());
```

Comment:
use System.currentTimeMillis() instead of LogUtil.getDateTimeStamp() and long as a type

Reply:
why long as a type?
and if the user wants to use something meaningful?

Reply:
does user wants something meaningful?